src/components/Chart.js

```diff
@@ -1,5 +1,64 @@
-import React from "react";
+import React, { Component } from "react";
+import ChartComponent, { Line } from "react-chartjs-2";
 
-const Chart = () => <div>Chart Component</div>;
+let values = [
+  {
+    date: "some_date_1",
+    value: 1
+  },
+  {
+    date: "some_date_2",
+    value: 2
+  },
+  {
+    date: "some_date_3",
+    value: 3
+  }
+];
+
+class Chart extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      chartData: {
```

Comment:
could you attach a link to documentation for chartData?
What are those fields and why is there so many backgroundColor?

Maybe some screenshot too :).

Reply:
https://www.chartjs.org/docs/latest/
  I deleted  unnecessary background colors